static/app/views/dashboardsV2/widgetBuilder/header.tsx

```diff
@@ -54,6 +54,15 @@ export function Header({
 
       <Layout.HeaderActions>
         <ButtonBar gap={1}>
+          <Button
+            title={t(
+              'How do you like the new widget builder? Submit feedback on GitHub.'
+            )}
+            href="https://github.com/getsentry/sentry/issues/new/choose"
+            target="_blank"
```

Comment:
should we just set "external" here?

Reply:
Good catch, I didn't know about the `external` prop but I've replaced it now, thanks!